src/npr/ont_pipeline.py

```diff
@@ -239,38 +340,38 @@ def read_flowcell_info(config, info_dict, base_path):
                     print(head[0].index('barcode_kit'))
         else:
             sys.exit("no json file, no final summary txt file found. exiting.")
+        
     print("flowcell = {}".format(info_dict["flowcell"]))
     print("kit = {}".format(info_dict["kit"]))
 
-    if (config['basecaller']=="guppy"):
-        if config['guppy_basecaller']['guppy_model'] is not None:
-            # record _full_ abnsolute path to model
-            info_dict['model'] = config['guppy_basecaller']['guppy_model']
-        else:
-            # infer model path name from flowcell and kit information using dictionary
-            modeldic = yaml.safe_load(
-                open(config['guppy_basecaller']['model_dictionary'])
-            )
-            info_dict['model'] = modeldic[info_dict['flowcell']][info_dict['kit']]
-            # modify model name if modification calling is desired
-            if config['guppy_basecaller']['guppy_mod'] is not None:
-                patt = r'(\w*)_(\w{3,4}\.cfg)'
-                repl = "modbases_" + config['guppy_basecaller']['guppy_mod']
-                info_dict['model'] = re.sub(patt, r'\1_{}_\2'.format(repl),info_dict['model'])
-
-    if (config['basecaller']=="dorado"):
-        if config['dorado_basecaller']['dorado_model'] is not None:
-            # record _full_ abnsolute path to model
-            info_dict['model'] = config['dorado_basecaller']['dorado_model']
-        else:
-            # default name of model derived from json (see above)
-            model_name = info_dict['model_def']
-            # brute force conversion to dorado
-            model_name = guppy2dorado(model_name)
-            info_dict['model'] = os.path.join(
-                config['dorado_basecaller']['model_directory'],
-                model_name
-            )
+    if info_dict["do_basecall"] == "do_basecall":
+        if (config['basecaller']=="guppy"):
+            if config['guppy_basecaller']['guppy_model'] is not None:
+                # record _full_ absolute path to model
+                info_dict['model'] = config['guppy_basecaller']['guppy_model']
+            else:
+                # infer model path name from flowcell and kit information using dictionary
+                modeldic = yaml.safe_load(
+                    open(config['guppy_basecaller']['model_dictionary'])
+                )
+                info_dict['model'] = modeldic[info_dict['flowcell']][info_dict['kit']]
+                # modify model name if modification calling is desired
+                if config['guppy_basecaller']['guppy_mod'] is not None:
```

Comment:
We should probably drop support for guppy soon, since it's not being used or tested anymore by us.

Reply:
Agree, this was discussed and need to be cleaned